scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,253 @@
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Command-Buffer Descriptor Type"
+name: $x_exp_command_buffer_desc_t
+base: $x_base_desc_t
+members: []
+--- #--------------------------------------------------------------------------
+type: typedef
+desc: "A value that identifies a command inside of a command-buffer, used for defining dependencies between commands in the same command-buffer."
+class: $xCommandBuffer
+name: $x_exp_command_buffer_sync_point_t
+value: uint32_t
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of Command-Buffer object"
+class: $xCommandBuffer
+name: "$x_exp_command_buffer_handle_t"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Create a Command-Buffer object"
+class: $xCommandBuffer
+name: CreateExp
+decl: static
+details:
+    - "Create a command-buffer object"
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: $x_device_handle_t
+      name: hDevice
+      desc: "[in] handle of the device object"
+    - type: "const $x_exp_command_buffer_desc_t*"
+      name: pCommandBufferDesc
+      desc: "[in][optional] CommandBuffer descriptor"
+    - type: "$x_exp_command_buffer_handle_t*"
+      name: phCommandBuffer
+      desc: "[out] pointer to Command-Buffer handle"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Increment the command-buffer object's reference count."
+class: $xCommandBuffer
+name: RetainExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Decrement the command-buffer object's reference count and delete the command-buffer object if the reference count becomes zero."
+class: $xCommandBuffer
+name: ReleaseExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Stop recording on a command-buffer object such that no more commands can be appended and make it ready to enqueue."
+class: $xCommandBuffer
+name: FinalizeExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
```

Comment:
We should copy the `-returns` from the core entry points and add invalid comment-

Reply:
I've updated the returns across all our functions and made sure they line up with the core entry points, thanks!